dev-resources/template/marketplace/1_secret.yml

```diff
@@ -0,0 +1,161 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
is this whole template just an abstract password generator for use in other templates? It seems like it asks for a path(s), and then CREATES the passwords in SSM? Just kind of skips anyone having to actually go in there i guess...